.gitignore

```diff
@@ -3,6 +3,8 @@
 # IDE Files
 .idea
 .idea/
+.vscode
+.vscode/
```

Comment:
i would prefer keeping the .vscode files. they are intended to be team wide settings that we can align upon - and are handy for vscode users. for example, i think we should use `prettier` formatting to not run into stylistic differences in code. have you used prettier before? any thoughts on using it?

Reply:
Ah, makes sense. I'm on Jetbrains, is prettier available for WebStorm?